### PR TITLE
Add Scylla Monitoring Stack landing page

### LINKED ISSUE #1345: Doc: Add link to Monitoring Git in the top menu
Add a link to the project github page
See https://prometheus.io/ for an example with the github icon

Maybe we can add this to the Scylla doc design.

### PR DESCRIPTION
**Pull request by lauranovich**
replaces https://github.com/scylladb/scylla-monitoring/pull/1442
FIXES #1345

Adds a landing page to the Scylla Monitoring stack docs site
Moves intro content to its own file
Adds mascot and topic boxes

BEFORE
Screenshot from 2021-06-16 14-35-53

AFTER
Screenshot from 2021-06-16 14-36-32

### CODE REVIEW

**Comment by lauranovich** [comment]
I was able to build a multiversion with no errors - it is just not pictured above

**Comment by lauranovich** [comment]
If possible we should backport this change

**Comment by tzach** [comment]
LGTM

File: docs/source/index.rst [hunk 1]
```diff
@@ -1,36 +1,108 @@
 Scylla Monitoring Stack
 =======================
 
+.. image:: scylla-monitor.png
+   :scale: 60%
+
```
Comment (dgarcia360) [inline docs/source/index.rst]:
````suggestion
.. title:: Scylla Monitoring Stack

.. raw:: html

  <div class="full-contents">

.. raw:: html

  <div class="main-title">
    <img class="main-title-logo" src="_static/img/scylla-monitor.png" />
    <div class="main-title-header">
      <h1>Scylla Monitoring Stack</h1>
      <h3>Complete NoSQL monitoring solution for your Scylla database clusters.</h3>
    </div>
  </div>
  
````

File: docs/source/index.rst [hunk 2]
```diff
@@ -1,36 +1,108 @@
...
 
-   User Guide <use-monitoring/index>
+   Introduction <intro>
    Download and Install <install/index>
+   User Guide <use-monitoring/index>
    Procedures <procedures/index>
    Troubleshooting <troubleshooting/index>
    Reference <reference/index>
 
 .. include:: /_common/monitor-description.rst
```
Comment (dgarcia360) [inline docs/source/index.rst]:
```suggestion
```

File: docs/source/index.rst [hunk 3]
```diff
@@ -1,36 +1,108 @@
...
-* :doc:`Procedures <procedures/index>`
-* :doc:`Troubleshooting <troubleshooting/index>`
-* :doc:`Reference <reference/index>`
-* `Scylla Monitoring Stack lesson <https://university.scylladb.com/courses/scylla-operations/lessons/admin-procedures-and-basic-monitoring/>`_ on Scylla University
+  Navigate to the Scylla Download Center
 
+.. raw:: html
 
-For older versions of Scylla Monitoring Stack Documentation see `here <https://docs.scylladb.com/operating-scylla/monitoring/>`_.
+  </div>
```
Comment (dgarcia360) [inline docs/source/index.rst]:
```suggestion
  </div></div>
```

**Review by dgarcia360 — CHANGES_REQUESTED** [review]

**Comment by dgarcia360** [comment]
After reviewing the PR, I'd add a few modifications so that the landing page look and feel is more aligned with other projects  (scylla-docs, sphinx-scylladb-theme):

image

To apply the changes:

1. Accept the suggestions proposed as comments.

2. In ``conf.py``, apply the following configuration changes:

- Under ``html_theme_options``, set ``'show_sidebar_index': False,``

- Replace the lines:

```
html_theme = 'sphinx_scylladb_theme'
html_style = ''
````
With:

```
html_theme = 'sphinx_scylladb_theme'
html_static_path = ['_static']
````

3. Under ``docs/source``, create a new directory ``_static/img``. Then, move ``scylla-monitor.png`` under ``_static/img/scylla-monitor.png``.

**Comment by lauranovich** [comment]
Now this is an improvement - thank you David for the review
Screenshot from 2021-06-17 16-42-20

**Comment by lauranovich** [comment]
@amnonh please merge this and backport as far as you can go.

**Comment by dgarcia360** [comment]
@lauranovich I don't recommend to backport yet - I'd wait until we release the new design to avoid doing it twice.

**Pull request merged**